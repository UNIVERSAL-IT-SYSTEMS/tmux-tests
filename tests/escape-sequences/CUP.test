#!/bin/sh

S=$(mktemp)
trap 'rm -f $S' 0 1 15

cat <<EOF >$S
for i in 0 10 20 30 40 50 60 70; do
	for j in 0 10 20; do
		printf "\033[\${j};\${i}H\${i},\${j}"
	done
done

printf "\033[25;81Hfoo"

printf "\033[26;Hbar"
printf "\033[;82Hbaz"

printf "\033[15;H15,"
printf "\033[;15H,15"

printf "\033[Hzero"

$TMUX_L wait -S tests
$TMUX_L wait never
EOF

$TMUX_L new -d -x 80 -y 25 -- sh $S
$TMUX_L wait tests\; capturep -ep -S- -E-
