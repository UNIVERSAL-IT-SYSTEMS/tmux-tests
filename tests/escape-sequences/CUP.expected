0,0      10,0      20,0      30,0      40,0      50,0      60,0      70,0
zero          ,15                                                              b
az






0,10     10,10     20,10     30,10     40,10     50,10     60,10     70,10





15,



0,20     10,20     20,20     30,20     40,20     50,20     60,20     70,20



                                                                               f
bar
